pages/about.html

```diff
@@ -4,5 +4,32 @@
 layout: about
 home: false
 ---
-
-<h2>About Content Goes Here</h2>
\ No newline at end of file
+<div class="aboutPage">
+  <div class="section">
+    <div class="infoContainer">
+      <h1>Who is Code for America?</h1>
+      <div class="text">
+        Code for America is a non-partisan, non-political 501 organization founded in 2009 to address the widening gap between the public and private sectors in their effective use of technology and design. Part of what they do is lead the Code for America Brigade Network where over 25,000 active members are committed to making their local communities better. Leveraging on the Code for America name and the tools they provide improve the way in which we work.
+      </div>
+      <p class="learnMore">
+        <a href="">Learn more about Code for America <i class="fas fa-long-arrow-alt-right"></i></a>
```

Comment:
Let's also reuse the style of the links we have on the main page (for links like Join us on Slack)

Can you also check with Joseph to see if we have the href url for both links on this page?
Might as well fill it in now!

Reply:
styles of links updated to match main page links. will check with Joseph on urls